pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -43,16 +40,15 @@ let
 
   inherit (lib.attrsets) removeAttrs;
 
-  pname = if args ? name && args.name != null then args.name else args.pname;
-  versionStr = optionalString (version != null) ("-" + version);
-  name = pname + versionStr;
+  name = args.name or "${args.pname}-${args.version}";
+  executableName = args.pname or args.name;
+  nameAttrs = lib.filterAttrs (key: value: builtins.elem key [ "name" "pname" "version" ]) args;
```

Comment:
Why do we need to remove these attrs again? A comment explaining this would be nice.

Reply:
they're not removed, they're singled out. we pass them into the final drv